QUESTIONS.md

```diff
@@ -1,5 +1,14 @@
 # Questions about Code, Node, Express, React
 
 What questions can you come up with?
-
+- More on React
+- More CSS please!
+- More HTML!
+- What is front end?
+ 
 ## Cohort Questions
+- When will profolio site be covered
+- Is there a syllabus we can see
+- Is there a way to know what assignments we still need to hand in
```

Comment:
I sent this very question to alex he said it was in progress.

Reply:
It would be helpful with the amount of projects we have!

Reply:
I am definitely interested to get more support for portfolio

Reply:
I completely forgot about it until mentioned earlier in class today!